packages/synapse-react-client/src/components/QueryWrapper/TableRowSelectionState.ts

```diff
@@ -0,0 +1,68 @@
+import { ColumnModel, Row } from '@sage-bionetworks/synapse-types'
```

Comment:
This file replaces the `useTableRowSelection`. It contains the table row selection atoms